x-pack/plugins/observability/public/pages/alerts/render_cell_value.tsx

```diff
@@ -52,13 +52,24 @@ export const getRenderCellValue = ({
   rangeFrom: string;
   setFlyoutAlert: (data: TopAlert) => void;
 }) => {
-  return ({ columnId, data, linkValues }: CellValueElementProps) => {
+  return ({ columnId, data, setCellProps }: CellValueElementProps) => {
     const { observabilityRuleTypeRegistry } = usePluginContext();
     const value = getMappedNonEcsValue({
       data,
       fieldName: columnId,
     })?.reduce((x) => x[0]);
 
+    useEffect(() => {
+      if (columnId === ALERT_DURATION) {
+        setCellProps({
+          style: {
+            textAlign: 'right',
+            paddingRight: '15px',
+          },
+        });
+      }
+    }, [columnId, data, setCellProps]);
```

Comment:
consider removing `data` as a dependency, because the styling doesn't need to be re-applied when the data changes (tested locally)

Reply:
I removed it thx!